### ISSUE TITLE
Code style changes

### ISSUE THREAD

**Issue by ericonr** [void-linux/xbps#276]
So, following @Duncaen suggestions, I'm taking a look at the code for xbps-query, to start off.

```
		{ "revdeps", required_argument, NULL, 'X' },
		{ "regex", no_argument, NULL, 0 },
		{ "fulldeptree", no_argument, NULL, 1 },
		{ "cat", required_argument, NULL, 2 },
```

**Comment by the-maldridge** [comment]
I like this idea, but I'd like to see it come with test cases to validate behavior before and after.  Ideally you'd do it as:

  * evaluate test coverage before, adding tests if necessary
  * make your changes
  * show that the tests continue to pass after adjusting from ints to enums

As you mentioned, raw ints are error proned, so care should be taken to not make the problem any worse than it already is.

**Comment by ericonr** [comment]
Fair enough. It seems XBPS has mainly integration tests instead of unit tests, so I will have to think about how to build them.
